questdb-rs/src/ingress/mod.md

```diff
@@ -0,0 +1,232 @@
+# Fast Ingestion of Data into QuestDB
+
+The `ingress` module implements QuestDB's variant of the
+[InfluxDB Line Protocol](https://questdb.io/docs/reference/api/ilp/overview/)
+(ILP) over HTTP and TCP. We recommend using HTTP because it has a number of
```

Comment:
Do you want to bother users with HTTP vs TCP from the very start, before showing them any code at all? why? Could we be opinionated in the begging and show this "decision point" only later? that's what I did with Java client: https://questdb.io/docs/clients/java_ilp/#transport-selection